### DISCUSSION TITLE
Sponsoring options should be more visible

### DISCUSSION THREAD

**Discussion by cshaa** [josdejong/mathjs#2204] [Ideas]
MathJS is a one-of-a-kind JavaScript library with 300k downloads weekly and Jos spends a lot of his time maintaining it, yet he only has 8 sponsors. I think there should be a “Sponsor” button in the readme, so that more people notice it.

**Comment by josdejong** [comment]
😄  that is very kind of you. I'm a terrible marketeer. Maybe the at the bottom of https://mathjs.org/ is a good place to add a sponsor button, alongside the like buttons?

**Reply by cshaa** [reply]
The bottom of mathjs.org sounds like a great place to put a sponsoring link. Also, putting one to the README would make it visible on NPM, which is also good imo. I'm thinking something like this:

![image](https://user-images.githubusercontent.com/1671665/117567376-72206d00-b0bc-11eb-88f0-05f632679a2a.png)

Example button: [![button](https://img.shields.io/badge/%20-Sponsor%20me%20on%20GitHub-orange)](https://github.com/sponsors/josdejong)

**Reply by josdejong** [reply]
Done 👍

**Reply by cshaa** [reply]
Great! I hope you get more Sponsors this way! 😁️

By the way, I don't know how Discussions work... Is there a way to close this? Or is it more like a forum that just stays open forever?

**Reply by josdejong** [reply]
No idea, it doesn't have a "Close" button or anything. I guess it just stays "open".

**Reply by gwhitney** [reply]
Well, it could be deleted, but I think it's nicer to keep the transcript. I will just lock it as the discussion appears to be done.
